platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/bool/EmfBooleanInput.java

```diff
@@ -2,15 +2,14 @@
 
 import com.softicar.platform.common.core.i18n.IDisplayString;
 import com.softicar.platform.common.core.interfaces.INullaryVoidFunction;
+import com.softicar.platform.common.core.interfaces.NullaryVoidFunctionList;
 import com.softicar.platform.dom.elements.checkbox.DomCheckbox;
 import com.softicar.platform.emf.attribute.input.IEmfInput;
-import java.util.Objects;
 import java.util.Optional;
-import java.util.function.Consumer;
 
 public class EmfBooleanInput extends DomCheckbox implements IEmfInput<Boolean> {
 
-	private INullaryVoidFunction callback = INullaryVoidFunction.NO_OPERATION;
+	private final NullaryVoidFunctionList callbacks = new NullaryVoidFunctionList();
```

Comment:
Hmm, we usually initialize fields in the CTOR.

Reply:
ok